sdk/keyvault/keyvault-keys/platform-matrix.json

```diff
@@ -9,7 +9,30 @@
         }
       },
       "TestType": "node",
-      "NodeTestVersion": "10.x"
+      "NodeTestVersion": "10.x",
+      "ServiceVersion": ""
+    },
+    {
+      "Agent": {
```

Comment:
Entries in `include` are also treated as matrix definitions, so you can shorten these new entries to:

```
    {
      "Agent": {
        "ubuntu-18.04": {
          "OSVmImage": "MMSUbuntu18.04",
          "Pool": "azsdk-pool-mms-ubuntu-1804-general"
        }
      },
      "TestType": "node",
      "NodeTestVersion": "10.x",
      "ServiceVersion": [ "7.0", "7.1" ]
    }
```

Reply:
The display name will show up like `ubuntu1804_node_10_70`, so if you want to override the display name, you can either add it at the top of the file:

```
{
  "displayNames": {
    "7.0": "service_version_7.0"
  }
}
```

Or override it with a grouping name:

```
    {
      "Agent": {
        "ubuntu-18.04": {
          "OSVmImage": "MMSUbuntu18.04",
          "Pool": "azsdk-pool-mms-ubuntu-1804-general"
        }
      },
      "TestType": "node",
      "NodeTestVersion": "10.x",
      "ServiceVersions": {
         "service_version_7.0": { "ServiceVersion": "7.0" },
         "service_version_7.1": { "ServiceVersion": "7.1" }
       }
    }
```

Reply:
[display name docs](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/job-matrix/README.md#displaynames)
[include docs](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/job-matrix/README.md#include)
[grouping/syntax docs](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/job-matrix/README.md#matrix-config-file-syntax)

Reply:
This is very nice - thank you, @benbp !